include/ck/tensor_operation/gpu/device/impl/device_grouped_mha_fwd_xdl_cshuffle_v2.hpp

```diff
@@ -600,6 +656,9 @@ struct DeviceGroupedMultiheadAttentionForward_Xdl_CShuffle_V2
 
         // for gridwise gemm check
         CGridDesc_M_N c_grid_desc_m_n_;
+
+        // raw data
+        std::array<std::vector<ck::index_t>, NumD0Tensor> d0s_nl_ns_lengths_strides_;
```

Comment:
Split this into two std::array<ck::index_t, NumD0Tensor> for last-dimension length and stride respectively, and renaming as `d0s_n_length` , `d0s_n_stride`, no need to use std::vector inside the std::array as these are only used for checking in `IsSupportedArgument()`

Reply:
done